### PR TITLE
RK-3587 - Track quit application for troubleshooting purposes

### PR DESCRIPTION
**Pull request by MickaelAlliel**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -189,14 +189,20 @@ function initAnalytics() {
     track("startup");
 }
 
+function quitApplication() {
+    track("quit-application", null, () => quitApplication());
```
Comment (amits1995) [inline src/index.ts]: app.quit

**Comment by amits1995** [comment]
Please test the behavior locally (also when offline)

**Comment by MickaelAlliel** [comment]
@amits1995 tested

**Pull request merged**